services/harmony/app/schemas/data-operation/0.20.0/data-operation-v0.20.0.json

```diff
@@ -0,0 +1,566 @@
+            "type": "string",
+            "format": "uri",
+            "qt-uri-protocols": [
+              "http",
+              "https"
+            ]
+          },
+          "stagingLocation": {
+            "name": "Result staging prefix",
+            "description": "An object store (S3) URL prefix under which services may elect to put their output.  Services must have write access to the Harmony staging bucket for the deployed environment to use this value.  The location will be unique per Harmony request but services are responsible for ensuring no name clashes occur within a single request.  The prefix will end in a \"/\" character",
+            "type": "string",
+            "examples": [
+              "s3://example-bucket/public/organization-name/service-name/1234567/"
+            ]
+          },
+          "user": {
+            "name": "Earthdata Login Username",
+            "description": "The name of the user on behalf of whom Harmony is acting",
+            "type": "string"
+          },
+          "accessToken": {
+            "name": "Earthdata Login Token",
+            "description": "The encrypted EDL token of the user on behalf of whom Harmony is acting",
+            "type": "string"
+          },
+          "client": {
+            "name": "Client ID",
+            "description": "An identifier indicating the client submitting the request",
+            "type": "string"
+          },
+          "version": {
+            "name": "Version number",
+            "description": "Identifies which schema version and Harmony callback protocol is being used",
+            "type": "string"
+          },
+          "isSynchronous": {
+            "name": "Synchronous request mode",
+            "description": "True if the request is going to be returned synchronously back to the end user. Note a backend service can still use a callback URL to indicate completion.",
+            "type": "boolean"
+          },
+          "requestId": {
+            "name": "Request identifier",
+            "description": "UUID to uniquely identify a request.",
+            "type": "string",
+            "format": "uuid"
+          },
+          "concatenate": {
+            "name": "Concatenate outputs",
+            "description": "True if the service should concatenate multiple input files into a single output file.",
+            "type": "boolean"
+          },
+          "average": {
+            "name": "Averaging method",
+            "description": "When provided the service should return the average of the inputs based on the averaging method.",
+            "examples": ["time", "area"],
+            "type": "string"
```

Comment:
Do we really want a string here or should this be an `enum`? I guess we want services to decide what they can handle, not the schema?